lib/network.py

```diff
@@ -1233,59 +1258,116 @@ def broadcast_transaction(self, transaction, callback=None):
 
     def get_history_for_scripthash(self, hash, callback=None):
         command = 'blockchain.scripthash.get_history'
-        invocation = lambda c: self.send([(command, [hash])], c)
+        invocation = lambda c: self._send([(command, [hash])], c)
 
-        return Network.__with_default_synchronous_callback(invocation, callback)
+        return Network.__with_default_synchronous_callback(
+            invocation,
+            callback)
 
     def subscribe_to_headers(self, callback=None):
         command = 'blockchain.headers.subscribe'
-        invocation = lambda c: self.send([(command, [True])], c)
+        invocation = lambda c: self._send([(command, [True])], c)
 
-        return Network.__with_default_synchronous_callback(invocation, callback)
+        return Network.__with_default_synchronous_callback(
+            invocation,
+            callback)
 
     def subscribe_to_address(self, address, callback=None):
```

Comment:
you might as well delete this method
(as it uses `blockchain.address.subscribe`)